cidc_api/config/settings.py

```diff
@@ -18,6 +18,10 @@
 TESTING = environ.get("TESTING") == "True"
 ## End application environment config
 
+## Configure Dev CFn
+CFNS_HTTP = environ.get("CFNS_HTTP")
```

Comment:
I might call this `DEV_FUNCTIONS_SERVER` or similar instead, since (as I understand it) that's what value will be stored in this environment variable.